docs/getting_started.md

```diff
@@ -4,7 +4,7 @@
 
 ## Prerequisites
 
-1. Python version >= 3.9.18 < 3.12
+1. Python version >= 3.10.18 < 3.12
```

Comment:
If we don't have specific reason to use exactly `3.10.18`, I would propose to remove the z-stream number from the version. (Previously the 3.9.18 were hard requirement, because there were some issue with lower z-stream version.)